e2e-test/src/main/java/org/alfresco/hxi_connector/e2e_test/CreateNodeTest.java

```diff
@@ -0,0 +1,170 @@
+
+    @Container
+    static final PostgreSQLContainer<?> postgres = DockerContainers.createPostgresContainerWithin(network);
+    @Container
+    static final GenericContainer<?> activemq = DockerContainers.createActiveMqContainerWithin(network);
+    @Container
+    static final GenericContainer<?> sfs = DockerContainers.createSfsContainerWithin(network);
+
+    @Container
+    private static final GenericContainer<?> transform_core_aio = DockerContainers.createTransformCoreAioContainerWithin(network)
+            .dependsOn(activemq)
+            .dependsOn(sfs);
+    @Container
+    static final GenericContainer<?> transform_router = DockerContainers.createTransformRouterContainerWithin(network)
+            .dependsOn(activemq)
+            .dependsOn(transform_core_aio)
+            .dependsOn(sfs);
+    @Container
+    static final GenericContainer<?> repository = createRepositoryContainer()
+            .dependsOn(postgres)
+            .dependsOn(activemq)
+            .dependsOn(transform_router)
+            .dependsOn(transform_core_aio)
+            .dependsOn(sfs);
+    @Container
+    static final WireMockContainer hxAuthServer = DockerContainers.createWireMockContainerWithin(network)
+            .withFileSystemBind("./src/main/resources/wiremock/hxinsight", "/home/wiremock", BindMode.READ_ONLY);
+    @Container
+    private static final GenericContainer<?> live_ingester = createLiveIngesterContainer()
+            .dependsOn(activemq)
+            .dependsOn(sfs)
+            .dependsOn(repository);
+    @Container
+    static final LocalStackContainer localStackServer = DockerContainers.createLocalStackContainerWithin(network);
+
+    @BeforeAll
+    @SneakyThrows
+    public static void beforeAll()
+    {
+        localStackServer.execInContainer("awslocal", "s3api", "create-bucket", "--bucket", BUCKET_NAME);
+    }
+
+    @Test
+    void testCreateFile()
+    {
+
+        Response acsResponse = given().auth().basic("admin", "admin")
+                .contentType("multipart/form-data")
+                .multiPart("filedata", new File("src/main/resources/test-files/Alfresco Content Services 7.4.docx"))
+                .when()
+                .post("http://" + repository.getHost() + ":" + repository.getFirstMappedPort() + "/alfresco/api/-default-/public/alfresco/versions/1/nodes/-my-/children")
+                .then()
+                .extract().response();
+
+        Assertions.assertEquals(201, acsResponse.statusCode());
+        Assertions.assertNotNull(acsResponse.jsonPath().get("entry.id"));
+
+        Response s3Response = given()
+                .contentType("application/xml")
+                .when()
+                .get("http://localhost:" + localStackServer.getFirstMappedPort() + "/test-hxinsight-bucket/")
+                .then()
+                .extract().response();
+
+        Assertions.assertEquals(200, s3Response.statusCode());
+        // String stringS3response = s3Response.asString();
```

Comment:
Do we need this commented out code here?
I suspect you wanted to add this assertion but we have an issue with S3 response but it could go in with a next PR.

Reply:
Yes, its exactly as you say